apps/tng-radio-hub/src/Components/pages/about/about.tsx

```diff
@@ -0,0 +1,67 @@
+import './about.module.css';
+
+/* eslint-disable-next-line */
+export interface AboutProps {}
+
+export function About(props: AboutProps) {
+    return (
+<div className="about">
+      <div className="backDrop">
+        <h1>About</h1>
+        <h2>We are an Indigenous broadcasting group</h2>
+        <h4>
+          supporting revitapzation and restoration of Tsilhqot’in language and
+          culture while building on the need to discuss relevant <br />{" "}
+          indigenous issues on a shareable platform to thousands across the
+          country. <br />
+          As the Tsilhqot’in Nation won Aboriginal Title in the Supreme Court of
+          Canada- <br />a first in Canadian history- we broadcast important
+          discussions relevant to First Nations, Inuit and Metis across Canada.
+        </h4>
+        <h2>Some of our governing principles</h2>
+
+        <h4>
+          • Promote the language, culture and customs of the Tsilhqot’in Nation{" "}
```

Comment:
Are you manually creating bullets? You should use html for unordered lists instead, if so.